<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-653N01I" failures="1" tests="6" name="Test" time="51.402" errors="0" timestamp="2021-12-04T10:20:30 IST">
  <testcase classname="tests.SearchTest" name="searchForANonExistingProduct" time="0.946"/>
  <testcase classname="tests.SearchTest" name="searchForAnExistingProduct" time="0.781"/>
  <testcase classname="tests.LoginTest" name="loginWithValidCredentials" time="1.877">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at tests.LoginTest.loginWithValidCredentials(LoginTest.java:41)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- loginWithValidCredentials -->
  <testcase classname="tests.LoginTest" name="loginWithInvalidUsernameAndInvalidPassword" time="1.992"/>
  <testcase classname="tests.LoginTest" name="loginWithInvalidUsernameAndValidPassword" time="1.546"/>
  <testcase classname="tests.LoginTest" name="loginWithValidUsernameAndInvalidPassword" time="1.552"/>
</testsuite> <!-- Test -->
